lib/hyrax/hirmeos/client.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+class Hyrax::Hirmeos::Client
+  attr_accessor :username, :password, :metrics_base_url, :token_base_url, :translation_base_url
+
+  def initialize(username, password, metrics_base_url, token_base_url, translation_base_url)
+    @username = username
+    @password = password
+    @metrics_base_url = metrics_base_url
+    @token_base_url = token_base_url
+    @translation_base_url = translation_base_url
+  end
+
+  def post_work(work)
+    con = id_translation_connection
+    con.post('/works', work.to_json)
```

Comment:
I think it is a little more readable if these two lines are combined.  What do you think?
```suggestion
    id_translation_connection.post('/works', work.to_json)
```